contrib/storage-hive/core/src/main/java/org/apache/drill/exec/expr/fn/HiveFunctionRegistry.java

```diff
@@ -48,8 +52,17 @@
 import org.apache.drill.shaded.guava.com.google.common.collect.ArrayListMultimap;
 import org.apache.drill.shaded.guava.com.google.common.collect.Sets;
 
-public class HiveFunctionRegistry implements PluggableFunctionRegistry{
-  static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(HiveFunctionRegistry.class);
+public class HiveFunctionRegistry implements PluggableFunctionRegistry {
+  private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(HiveFunctionRegistry.class);
+
+  /**
+   * Map for renaming Hive UDFs whose names satisfy the predicate in the key by names from the map value.
+   */
+  private static final Map<Predicate<String[]>, String[]> FUNCTION_REPLACE_MAP = ImmutableMap.<Predicate<String[]>, String[]> builder()
```

Comment:
Please add better description.

Reply:
@vvysotskyi Try to simplify the logic for `FUNCTION_REPLACE_MAP`. Possibly try to get rid from `String[]` (you can add all function names to this Map), so in result it can be similar to 
https://github.com/apache/drill/blob/master/logical/src/main/java/org/apache/drill/common/expression/fn/FunctionReplacementUtils.java#L42

Reply:
Thanks, reworded a description and simplified the map to use target function names as keys and predicates for checking as values.